007_smoothed_Laplace/smoothed_laplace.ipynb

```diff
@@ -561,14 +569,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We see that the posterior mean is driven to zero at most of the locations, and the uncertainty there is very small. This shows the effectiveness of using a Laplace prior for such a signal."
+    "The posterior mean accurately reconstructs the zero values, such as at index 20. However, it exhibits noticeable peaks at the transition points where the exact signal shifts from zero to one and back to zero, such as at around index 56. The credibility interval is generally narrow across most of the domain, indicating low uncertainty, but it broadens significantly around the transition points of the exact signal, reflecting increased uncertainty in these areas.\n",
```

Comment:
Please mention around here that the vertical/horizontal black lines in the trace plots are the true parameter values for comparison.

Reply:
Changed to:
> "...we plot the trace and mean of the posterior samples, with the exact signal included in black for comparison."